Applications/spire/source/charting/charting_technicals_panel.cpp

```diff
@@ -0,0 +1,210 @@
+#include "spire/charting/charting_technicals_panel.hpp"
+#include <QHBoxLayout>
+#include <QVBoxLayout>
+#include "spire/spire/dimensions.hpp"
+#include "spire/spire/technicals_model.hpp"
+
+using namespace Nexus;
+using namespace Spire;
+
+ChartingTechnicalsPanel::ChartingTechnicalsPanel(TechnicalsModel& model)
+    : m_model(model) {
+  m_item_delegate = new CustomVariantItemDelegate(this);
```

Comment:
You can not use a model before loading it.

Reply:
So should it be the responsibility of the creator of the model to load it, or should every class that uses it assume that it hasn't been loaded?

Reply:
Review existing solutions, consider pros and cons, think carefully and then propose a solution.

Reply:
If load() is loosely defined as "get the model ready" then the creator should be responsible for it.